packages/web/pages/login.tsx

```diff
@@ -1,62 +1,50 @@
 import { Flex, Image, SimpleGrid, Text } from '@metafam/ds';
-import React from 'react';
+import React, { useState } from 'react';
 
 import { MetaLink } from '../components/Link';
-import BackgroundImage from '../public/images/background.png';
+import { RegisterPlayer } from '../components/RegisterPlayer';
+import { WelcomePlayer } from '../components/WelcomePlayer';
+import BackgroundImage from '../public/images/login-background.png';
 import MetaGameImage from '../public/images/metagame.png';
-import PlayersImage from '../public/images/players.png';
 
-const Login: React.FC = () => (
-  <Flex
-    backgroundSize="cover"
-    backgroundImage={`url(${BackgroundImage})`}
-    width="100vw"
-    height="100vh"
-    padding="3rem"
-    flexDirection="column"
-    alignItems="center"
-  >
-    <SimpleGrid columns={3} alignItems="center" width="100%">
-      <MetaLink fontFamily="heading" href="" color="#79F8FB">
-        How to play MetaGame
-      </MetaLink>
-      <Flex justifyContent="center" alignItems="center">
-        <Image src={MetaGameImage} />
-      </Flex>
-      <Flex alignItems="center" justifyContent="flex-end">
-        <Text fontFamily="heading" color="#F6F8F9">
-          Already a Player?{' '}
-          <MetaLink fontFamily="heading" href="" color="#79F8FB">
-            Sign in
-          </MetaLink>
-        </Text>
-      </Flex>
-    </SimpleGrid>
+const Login: React.FC = () => {
+  const [step, setStep] = useState(0);
+  return (
     <Flex
-      alignItems="center"
-      justifyContent="center"
+      backgroundSize="cover"
+      backgroundImage={`url(${BackgroundImage})`}
+      width="100vw"
+      height="100vh"
+      padding="3rem"
       flexDirection="column"
-      flex={1}
-      maxWidth="45rem"
+      alignItems="center"
     >
-      <Image src={PlayersImage} />
-      <Text
-        fontSize="3xl"
-        fontFamily="mono"
-        fontWeight="bold"
-        color="#FFFFFF"
-        m={5}
-      >
-        Become a Player
-      </Text>
-      <Text fontSize="lg" fontFamily="body" color="#FFFFFF" textAlign="center">
-        MetaGame is an idea we can build a new world, a layer atop of the old
-        one. A more collaborative, transparent & caring world. A world in which
-        self-interest is better aligned with the common good & the ones creating
-        value are more directly rewarded.
-      </Text>
+      <SimpleGrid columns={3} alignItems="center" width="100%">
+        <MetaLink fontFamily="heading" href="" color="cyan.400">
+          How to play MetaGame
+        </MetaLink>
+        <Flex justifyContent="center" alignItems="center">
+          <Image src={MetaGameImage} />
+        </Flex>
+        <Flex alignItems="center" justifyContent="flex-end">
+          <Text fontFamily="heading" color="offwhite.50">
```

Comment:
Use Heading component

Reply:
As mention above default fontSize of <Heading /> is too large. These are Text components with a different font. 
We could possibly set the fontFamily "heading" as the default though